### PR TITLE
Cleanup code for Hasura event triggers, SQS, and related CI

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13275

This PR:
- removes the SQS + Lambda build and cleanup workflows for Hasura event trigger handling
- removes the helpers for those workflows
- removes the associated code from the VZ API Flask app
- removes the remaining files from the `atd-events` folder since we no longer have any Hasura event triggers

## Testing
**URL to test:** 
n/a

**Steps to test:**
Not much to test here. We should see two less GH action workflows when this merges into `master` branch.

---
#### Ship list
- [ ] ~Run migration steps in readme in /atd-vzd/ if needed~
- [x] Code reviewed
- [x] Product manager approved
#### After merge
- [ ] Remove `HASURA_TRIGGER_API_KEY` from `ZAPPA_SETTINGS` and task definitions for staging and prod VZ clusters

### CODE REVIEW

File: atd-cr3-api/server.py [hunk 1]
```diff
@@ -8,13 +8,11 @@
 import boto3
 import os
 import requests
-import hashlib
```
Comment (mddilley) [inline atd-cr3-api/server.py:11]: This was only used in verifying the secret string called `HASURA_TRIGGER_API_KEY ` that was used to know that a message was coming from our Hasura engine to our API.

**Review by frankhereford — APPROVED** [review]
So much delete! This is great, thank you for both doing this and for your big role in making it possible at all. 🙏

**Review by roseeichelmann — APPROVED** [review]
Thanks for taking care of this Mike :partying_face:

**Review by chiaberry — APPROVED** [review]
awesome clean up!!

**Review by patrickm02L — APPROVED** [review]
Good to go! 🧹

**Comment by johnclary** [comment]
i'm always envious of the person who gets to delete code—nice!

**Pull request merged**